application.py

```diff
@@ -1,6 +1,7 @@
 from flask import Flask, request, jsonify
 from flask_sqlalchemy import SQLAlchemy
 from flask_marshmallow import Marshmallow
+import numpy as np
```

Comment:
We may need to add this to requirements.txt since this is a new library.

Reply:
Yes.  We can test this when we deploy.  I didn't have to install the library explicitly on my local pc so I'm thinking that it may not be needed on requirements.txt